tensorflow/tools/ci_build/osx/arm64/tensorflow_as_build_nightly.Jenkinsfile

```diff
@@ -0,0 +1,253 @@
+/*
+Copyright 2023 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+pipeline {
+    agent none
+
+    stages {
+        stage("Build Tensorflow") {
+            parallel {
+                stage("Python 3.8") {
+                    agent {
+                        label "nightly-build"
+                    }
+                    environment {
+                        PYENV_ROOT="$HOME/.pyenv"
+                        PATH="$PYENV_ROOT/shims:/opt/homebrew/bin/:$PATH"
+
+                    }
+                    steps {
+                        dir('tensorflow') {
+
+                            sh '''
+                                pyenv init -
+                                pyenv global 3.8.13
+                            '''
+
+                            sh 'python --version'
+
+                            git branch: "nightly",
+                                url: "https://github.com/tensorflow/tensorflow.git"
+
+                            sh '''
+                                pip install --upgrade pip
+                                pip install -r ./tensorflow/tools/ci_build/release/requirements_mac.txt
+                                pip install Pillow
+                                python tensorflow/tools/ci_build/update_version.py --nightly
+                            '''
+
+                            sh '''
+                                /opt/homebrew/bin/bazel --bazelrc="${WORKSPACE}/tensorflow/tools/ci_build/osx/arm64/.macos.bazelrc" build \
+                                --action_env PYTHON_LIB_PATH="/Users/admin/.pyenv/versions/3.8.13/lib/python3.8/site-packages" \
+                                --action_env MACOSX_DEPLOYMENT_TARGET=11.3 \
```

Comment:
We can change the version to 11.0 and move this flag to macos.bazelrc.

Reply:
done